K8s/manifest/jaeger.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jaeger
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: jaeger
+  template:
+    metadata:
+      labels:
+        app: jaeger
+    spec:
+      containers:
+        - name: jaeger
+          image: jaegertracing/all-in-one:1.43.0
+          ports:
+            - containerPort: 5775
+              protocol: UDP
+            - containerPort: 6831
+              protocol: UDP
+            - containerPort: 6832
+              protocol: UDP
+            - containerPort: 5778
+            - containerPort: 16685
+            - containerPort: 16686
+            - containerPort: 14250
+            - containerPort: 14268
+            - containerPort: 14269
+            - containerPort: 9411
+            - containerPort: 4317
+              protocol: TCP
+            - containerPort: 4318
+              protocol: TCP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: jaeger
+spec:
+  selector:
+    app: jaeger
+  ports:
+    - name: port-5775
+      protocol: UDP
+      port: 5775
+      targetPort: 5775
+    - name: port-6831
+      protocol: UDP
+      port: 6831
+      targetPort: 6831
+    - name: port-6832
+      protocol: UDP
+      port: 6832
+      targetPort: 6832
+    - name: port-5778
+      protocol: TCP
+      port: 5778
+    - name: port-16685
+      protocol: TCP
+      port: 16685
+    - name: port-16686
+      protocol: TCP
+      port: 16686
+    - name: port-14250
+      protocol: TCP
+      port: 14250
+    - name: port-14268
+      protocol: TCP
+      port: 14268
+    - name: port-14269
+      protocol: TCP
+      port: 14269
+    - name: port-9411
+      protocol: TCP
+      port: 9411
+    - name: otlp-grpc
+      port: 4317
+    - name: otlp-http
+      port: 4318
+  type: LoadBalancer
```

Comment:
We should revert this at some point. Maybe let's comment this out.

Reply:
I've pushed a new version of this file where we have 2 services and just port 16686 is as LoadBalancer the other one (14250, 4317, 4318) are as ClusterIP (only internal)